oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -340,6 +343,23 @@ public AccessToken refreshAccessToken() throws IOException {
     HttpRequest request = requestFactory.buildPostRequest(new GenericUrl(tokenServerUri), content);
     request.setParser(new JsonObjectParser(jsonFactory));
 
+    request.setIOExceptionHandler(new HttpBackOffIOExceptionHandler(new ExponentialBackOff()));
+    request.setUnsuccessfulResponseHandler(
+        new HttpBackOffUnsuccessfulResponseHandler(new ExponentialBackOff()).setBackOffRequired(
```

Comment:
Here's a comparison of parameters with those used many places for gRPC calls in google-cloud-java. These seem to me to be in the same ballpark. I might generally expect longer settings for this use case since, in a typical scenario, it would be critical to all following operations.

Value | google-cloud-java standard | `ExponentialBackOff` default
------------ | ------------- | -------------
initial delay | 0.1 sec | 0.5 sec
base delay multiplier | 1.3 | 1.5
random delay multiplier | 0-1 | 0.5-1.5
max delay | 1 min | 1 min
initial timeout | 20 sec | none
timeout multiplier | 1 | none
max timeout | 20 sec | none
max total | 10 min | 15 min